package.json

```
@@ -92,7 +91,7 @@
     "commitizen": "^4.2.4",
     "cpx": "^1.5.0",
     "cross-env": "^7.0.3",
-    "cspell": "^4.2.8",
+    "cspell": "6",
```

```suggestion
    "cspell": "^6.19.2",
```

Could you please update this to follow the semver standards?